src/v/cluster/members_manager.cc

```diff
@@ -440,59 +441,64 @@ auto members_manager::dispatch_rpc_to_leader(
       std::forward<Func>(f));
 }
 
-ss::future<result<join_reply>>
-members_manager::handle_join_request(model::broker broker) {
-    using ret_t = result<join_reply>;
-    vlog(clusterlog.info, "Processing node '{}' join request", broker.id());
+ss::future<result<join_node_reply>>
+members_manager::handle_join_request(join_node_request const& req) {
+    using ret_t = result<join_node_reply>;
+    vlog(clusterlog.info, "Processing node '{}' join request", req.node.id());
```

Comment:
maybe we can add some more context from `join_node_request` to that log message f.e. version

Reply:
Added version here.